### PR TITLE
chore: add row_limit for text generators

### PR DESCRIPTION
**Pull request by sycai**
This helps prevent unexpected large amount of data being processed by LLM

### CODE REVIEW

**Comment by shobsi** [comment]
> This helps prevent unexpected large amount of data being processed by LLM

Is this to server a customer feedback or to support our test infra?

File: bigframes/ml/llm.py [hunk 1]
```diff
@@ -598,6 +598,8 @@ class TextEmbeddingGenerator(base.BaseEstimator):
         connection_name (str or None):
             Connection to connect with remote service. str of the format <PROJECT_NUMBER/PROJECT_ID>.<LOCATION>.<CONNECTION_ID>.
             If None, use default connection in session context.
+        row_limit (int or None):
```
Comment (chelsea-lin) [inline bigframes/ml/llm.py:601]: nit: `max_cols`, which is used in other pandas methods.

File: bigframes/ml/llm.py [hunk 2]
```diff
@@ -598,6 +598,8 @@ class TextEmbeddingGenerator(base.BaseEstimator):
...
+            The maximum number of rows that this model can process.
```
Comment (chelsea-lin) [inline bigframes/ml/llm.py:602]: nit: `The maximum number of rows that this model can process per call`. The model can be called multiple times.

**Review by chelsea-lin — APPROVED** [review]
LGTM with 2 nit comments.

File: bigframes/ml/llm.py (hunk 2 above)
Comment (sycai) [reply bigframes/ml/llm.py:602]: SG!

File: bigframes/ml/llm.py (hunk 1 above)
Comment (sycai) [reply bigframes/ml/llm.py:601]: Done

File: bigframes/ml/llm.py [hunk 3]
```diff
@@ -598,8 +598,8 @@ class TextEmbeddingGenerator(base.BaseEstimator):
         connection_name (str or None):
             Connection to connect with remote service. str of the format <PROJECT_NUMBER/PROJECT_ID>.<LOCATION>.<CONNECTION_ID>.
             If None, use default connection in session context.
-        row_limit (int or None):
-            The maximum number of rows that this model can process.
+        max_rows (int or None):
+            The maximum number of rows that this model can process per call.
```
Comment (TrevorBergeron) [inline bigframes/ml/llm.py:602]: My feeling is this is better as a bigframes-wide setting. Alternatively, if we are just focusing on sem-join cardinality explosion, we can apply a narrow limit just for that case

**Comment by chelsea-lin** [comment]
Added `do not merge` for further API reviewing.

**Pull request closed without merging**